### DISCUSSION TITLE
Model Suggestions

### DISCUSSION THREAD

**Discussion by GemmaTuron** [ersilia-os/ersilia#694] [Ideas]
This is an open discussion where new models can be suggested!
If the community finds them interesting and aligned to Ersilia's topics of interest, we will include them in the TODO list [here](https://airtable.com/shroQLlkcmDcC0xzm).

At this moment, we are looking for models that:
🦠 Relate to infectious diseases
💊 Tackle any stage of the drug discovery cascade
🐍 Are written in Python (unless the model contributor is expert in another language and feels ready to tackle integration)
📝 Are published in the scientific literature at least as a pre print
🔓Are Open Source

Please, when suggesting a new model, provide a short description of it, why is it relevant to Ersilia and the links to the publication and code

**Comment by GemmaTuron** [comment]
I can start by suggesting a model published in [Nature Chemical Biology](https://www.nature.com/articles/s41589-023-01349-8) a few days ago: Deep learning-guided discovery of an antibiotic targeting Acinetobacter baumannii

Using the Chemprop network, the authors (original developers of Chemprop) develop a model to predict novel A.Baumanni inhibitors using a dataset of 7500 in vitro experimental datapoints and led to the discovery of a new drug candidate with good in vitro and in vivo activity.
The code for the model is available in [this fork](https://github.com/GaryLiu152/chemprop_abaucin) of Chemprop (MIT License)

**Comment by ZakiaYahya** [comment]
> This is an open discussion where new models can be suggested! If the community finds them interesting and aligned to Ersilia's topics of interest, we will include them in the TODO list [here](https://airtable.com/shroQLlkcmDcC0xzm).
> 
> At this moment, we are looking for models that: 🦠 Relate to infectious diseases 💊 Tackle any stage of the drug discovery cascade 🐍 Are written in Python (unless the model contributor is expert in another language and feels ready to tackle integration) 📝 Are published in the scientific literature at least as a pre print 🔓Are Open Source
> 
> Please, when suggesting a new model, provide a short description of it, why is it relevant to Ersilia and the links to the publication and code

I'm suggesting a model: SMILES Transformer: Pre-trained Molecular Fingerprint for Low Data Drug Discovery. You can find the it's published paper [here](https://arxiv.org/abs/1911.04738)
SMILES Transformer extracts molecular fingerprints from string representations of chemical molecules. The transformer learns latent representation that is useful for various downstream tasks through autoencoding task. They pre-train Transformers with unlabeled SMILES to learn their representations and show the potential of textbased models compared to baseline models including graph convolutions. They used Chembl24 dataset for this purpose.
The github code for the model is available [here](https://github.com/DSPsleeporg/smiles-transformer) (MIT License)

**Reply by DhanshreeA** [reply]
This looks like a good model, please add it to the [model suggestion form](https://airtable.com/shroQLlkcmDcC0xzm) @ZakiaYahya

**Reply by ZakiaYahya** [reply]
Hi @DhanshreeA
Sure, i'll add it in the suggestion list.
Thanks.

**Reply by ZakiaYahya** [reply]
Done

**Comment by HellenNamulinda** [comment]
> This is an open discussion where new models can be suggested! If the community finds them interesting and aligned to Ersilia's topics of interest, we will include them in the TODO list [here](https://airtable.com/shroQLlkcmDcC0xzm).
> 
> At this moment, we are looking for models that: microbe Relate to infectious diseases pill Tackle any stage of the drug discovery cascade snake Are written in Python (unless the model contributor is expert in another language and feels ready to tackle integration) memo Are published in the scientific literature at least as a pre print unlockAre Open Source
> 
> Please, when suggesting a new model, provide a short description of it, why is it relevant to Ersilia and the links to the publication and code

I would like to suggest **miDruglikeness:** Subdivisional Drug-Likeness Prediction Models Using Active Ensemble Learning Strategies .
miDruglikeness is a composite model that provides predictions on the drug-likeness of compounds at different stages of drug discovery and development. It can predict in vivo ability, IND (investigational new drug) ability and the likelihood of market approval for compounds.

The paper was published in Biomolecules 2023 and it can be accessed [here]( https://doi.org/10.3390/biom13010029)
Using graph neural networks, the authors developed three subdivisional models. These models were designed to individually predict the capacity of a compound to enter in vivo testing, clinical trials, and market approval stages. The authors also proposed a strategy that combined active learning and ensemble learning to improve the quality of the models. All training datasets were obtained from the ZINC15 subsets. These models achieved satisfactory performance on internal test datasets and four self-collected external test datasets.
The models also demonstrated great virtual screening capabilities when evaluated on the DEKOIS 2.0 benchmark dataset.

The code and datasets used  are available in [this repo]( https://github.com/tytcc/miDruglikeness) (No License details provided)

**Reply by GemmaTuron** [reply]
Good one @HellenNamulinda !

Go ahead and fill in the [model suggestion form](https://airtable.com/shroQLlkcmDcC0xzm)

**Comment by emmakodes** [comment]
### Model: 
GNN-MTB: An Anti-Mycobacterium Drug Virtual Screening Model Based on Graph Neural Network

### Model Description:
The paper implements and builds an anti-Tuberculosis inhibitor prediction model based on Curriculum Learning (CL) and Graph Neural Network (GNN) methods. They used the data of 10,789 compounds collected from the open-access drug experiment data (ChEMBL) to carry out the training and validation of this virtual screening model for anti-tuberculosis drugs

### Publication:
https://manu44.magtech.com.cn/Jwk_infotech_wk3/EN/abstract/abstract5544.shtml

Github Repo:
https://github.com/gu-yaowen/GNN-MTB

### Licence:
None

**Reply by GemmaTuron** [reply]
Let's add it to the model suggestion list Emmanuel, thanks!

**Reply by bankthammavong** [reply]
Good,thanks

**Comment by ZakiaYahya** [comment]
I'm suggesting a model: _Virtual screening of DrugBank database for hERG blockers using topological Laplacian-assisted AI models_. You can find the it's published paper [here](https://www.sciencedirect.com/science/article/abs/pii/S0010482522011994)

The potential blockade of the potassium ion channel can lead to fatal disorders and/or long QT syndrome. Numerous drugs have been withdrawn due to their severe hERG-cardiotoxicity. Assessing the hERG blockade activity in the early stages of drug discovery is of utmost importance. In this paper, their focus specifically lies in evaluating the hERG-cardiotoxicity of compounds collected in the DrugBank database, given that many of these compounds have already been approved for therapeutic treatments or exhibit high potential to become drugs. They design accurate and robust classifiers for blockers/non-blockers and then build regressors to quantitatively analyze the binding potency of the DrugBank compounds on the hERG channel. Molecular sequences are embedded with two natural language processing (NPL) methods, namely, autoencoder and transformer.

The github code for the model is available [here](https://github.com/weilabmsu/herg-prediction)  with No License details provided.

**Reply by DhanshreeA** [reply]
This looks pretty good to me, what do you think @GemmaTuron?

**Comment by ZakiaYahya** [comment]
I'm suggesting a model: _Accurate ADMET Prediction with XGBoost_. You can find it's published paper [here](https://arxiv.org/abs/2204.07532)

ADMET (absorption, distribution, metabolism, excretion, and toxicity) properties play a crucial role in drug discovery as they determine the effectiveness and safety of drugs. In this study, they utilized a combination of features such as fingerprints and descriptors, along with a tree-based machine learning model called extreme gradient boosting, to accurately predict ADMET properties. This model demonstrated excellent performance in the Therapeutics Data Commons ADMET benchmark group. Among the 22 tasks,  this model achieved the top rank in 18 tasks and was among the top 3 in 21 tasks. The trained machine learning models have been integrated into ADMETboost, a publicly accessible web server available at https://ai-druglab.smu.edu/admet.

The checkpoints are not available but they have provided with the code `model.py` to train model and get predictions. The github code for the model is available [here](https://github.com/smu-tao-group/ADMET_XGBoost) with `GPL-3.0` License provided.
